src/app/member/[memberIdentifier]/dashboard/page.tsx

```diff
@@ -17,6 +35,31 @@ export default async function MemberDashboardPage(props: { params: Promise<{ mem
         return <AlertNotFound title="Member was not found" message="no such member exists" />
     }
 
+    const studies = await fetchStudiesForCurrentMemberAction()
+
+    const rows = studies.map((study) => (
+        <TableTr key={study.id}>
```

Comment:
Took some inspiration from you @nathanstitt now this is also a server component 😄